ambry-api/src/main/java/com/github/ambry/clustermap/VcrClusterSpectator.java

```diff
@@ -18,9 +18,9 @@
 
 
 /**
- * {@link ClusterSpectator} adds itself as a spectator of a cluster and registers some cluster change listeners.
+ * {@link VcrClusterSpectator} adds itself as a spectator of a cluster and registers some cluster change listeners.
  */
-public interface ClusterSpectator extends InstanceConfigChangeListener, LiveInstanceChangeListener {
+public interface VcrClusterSpectator extends InstanceConfigChangeListener, LiveInstanceChangeListener {
```

Comment:
nit: does it need to be called `VcrClusterSpectator` instead of `ClusterSpectator` ? It doesn't look to have any vcr specific methods.

Reply:
Based on Casey's suggestion, we will change VcrClusterSpectator to not implement helix interfaces and instead deal with ambry defined VCR listener interfaces. So this is will be a VCR specific spectator.